src/neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -122,7 +122,7 @@ partial class ApplicationEngine
         /// The <see cref="InteropDescriptor"/> of System.Runtime.GetNotifications.
         /// Gets the notifications sent by the specified contract during the execution.
         /// </summary>
-        public static readonly InteropDescriptor System_Runtime_GetNotifications = Register("System.Runtime.GetNotifications", nameof(GetNotifications), 1 << 8, CallFlags.None);
+        public static readonly InteropDescriptor System_Runtime_GetNotifications = Register("System.Runtime.GetNotifications", nameof(GetNotifications), 1 << 12, CallFlags.None);
```

Comment:
Isn't it a hardfork?

Reply:
Yes, i think so

Reply:
Full sync mainnet, storage doesn't fork.